lectures/equalizing_difference.md

```diff
@@ -518,12 +460,4 @@ Let's compute $\frac{\partial \phi}{\partial R}$ and evaluate it numerically at
 ϕ_R_func(D_value, γ_h_value, γ_c_value, R_value, T_value, w_h0_value)
 ```
 
-+++ {"tags": []}
-
-We find that raising the gross interest rate $R$ increases the initial college wage premium $\phi$, as we did with our graphical analysis earlier
-
-
-
-```{code-cell} ipython3
-
-```
\ No newline at end of file
+We find that raising the gross interest rate $R$ increases the initial college wage premium $\phi$, as we did with our graphical analysis earlier
```

Comment:
"as we did with our graphical analysis earlier" -> "in line with our earlier graphical analysis"